aptos-node/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "aptos-node"
 description = "Aptos node"
-version = "1.6.0"
+version = "1.7.0"
```

Comment:
We don't bump patch version?

Reply:
No -- we've typically only done major release versions. But if we can automate and/or test this, minor versions make sense to me 😄